packages/express/index.ts

```diff
@@ -0,0 +1,51 @@
+import type { RequestHandler } from 'express'
+import { Router } from 'express'
+import type { KopflosConfig } from '@kopflos-cms/core'
+import absolutUrl from 'absolute-url'
+import Kopflos from '@kopflos-cms/core'
+import rdfHandler from '@rdfjs/express-handler'
+import factory from '@zazuko/env-node'
+import onetime from 'onetime'
+import { match, P } from 'ts-pattern'
+import asyncMiddleware from 'middleware-async'
+
+export default (options: KopflosConfig): RequestHandler => {
+  const kopflos = new Kopflos(options)
+
+  return Router()
+    .use(onetime(async (req, res, next) => {
+      if (options.apiGraphs) {
+        await Kopflos.fromGraphs(kopflos, ...options.apiGraphs)
+      } else {
+        return next(new Error('No API graphs configured. In future release it will be possible to select graphs dynamically.'))
```

Comment:
For a moment I thought to use a default such as `/kopflos` but this may not be ideal. 

In the future we could actually query to discover graphs somehow